packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -67,14 +69,42 @@ export default class ContentSwitcher extends React.Component {
     );
   }
 
+  /**
+   * A "ring buffer" function that takes an array and depending on an ArrowRight
+   * or ArrowLeft key input loops from last index to first or first index to last.
+   */
+  handleOnKeyDown = (key, index, arrayLength) => {
```

Comment:
Typically `handle*` named methods will be used for responding to events from the environment. e.g. `handleOnClick`. If it's a helper method, might help to name it according to its purpose, for example: `getNextIndex`